wherehows-dao/src/main/java/wherehows/dao/DaoFactory.java

```diff
@@ -35,4 +35,20 @@ public DaoFactory(EntityManagerFactory entityManagerFactory) {
   public DatasetClassificationDao getDatasetClassificationDao() {
     return new DatasetClassificationDao(entityManagerFactory);
   }
+
+  public DictDatasetDao getDictDatasetDao() {
```

Comment:
Dataset.java(same thing as DictDatasetClass) class right now is used in frontend and backend in a lot of places. Change DictDataset.java to Dataset.java means i need to merge those two classes, how about i do the refactoring work in another PR?